dist/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.3/css/all.css"
+      integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk"
+      crossorigin="anonymous"
+    />
+    <title>Lol</title>
+  <script defer src="bundle.js"></script></head>
+  <body>
+    <div id="content" class="container-fluid pt-5">
+      <!-- navbar -->
+      <nav class="navbar navbar-expand navbar-light bg-light fixed-top">
+        <div class="container-fluid">
+          <a class="navbar-brand lobster me-5" href="#">LOL</a>
+          <button
+            class="navbar-toggler"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarText"
+            aria-controls="navbarText"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarText">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+              <li class="nav-item">
+                <a class="nav-link home" aria-current="page" href="#">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link menu" href="#">Menu</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link contact" href="#">Contact</a>
+              </li>
+            </ul>
+            <span class="navbar-text d-none d-sm-block">
+              We love Lasagna <i class="fa fa-heart"></i
+            ></span>
+          </div>
+        </div>
+      </nav>
+    </div>
```

Comment:
[OPTIONAL] It **appends** HTML elements to the **< div >** with **id=content** when the page is loaded.

Reply:
Hello, I don't understand this optional point:[OPTIONAL] It appends HTML elements to the < div > with id=content when the page is loaded.

Reply:
Also, I already have set the development mode is it really necessary to add a script?